scenarios/generate-synthetic-data/non-adversarial-simulator/summarization/simulate_and_evaluate_summarization.ipynb

```diff
@@ -0,0 +1,181 @@
+    "    },\n",
+    "    {\n",
+    "        \"name\": \"Jane Doe\",\n",
+    "        \"chatbot_name\": \"AI Chatbot\",\n",
+    "        \"filename\": \"sales_report.txt\",\n",
+    "        \"file_content\": \"The sales team is doing well. The sales team is meeting its targets. The sales team is increasing its revenue. The sales team is increasing its market share. The sales team is increasing its profits. The sales team is expanding into new markets. The sales team is launching new products. The sales team is opening new stores. The sales team is hiring new employees. The sales team is investing in new technology.\",\n",
+    "    },\n",
+    "]\n",
+    "template = Simulator.get_template(\"summarization\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Run the simulator"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "outputs = await simulator.simulate_async(\n",
+    "    template,\n",
+    "    parameters=template_params,\n",
+    "    max_conversation_turns=2,\n",
+    "    api_call_delay_sec=10,\n",
+    "    max_simulation_results=10,\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Print the outputs"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "for line in outputs:\n",
+    "    print(json.dumps(line, indent=2))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Store outputs in file for eval"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "eval_name = \"conv-oai-summarization-eval\"\n",
+    "file_name = f\"\\{eval_name}.jsonl\"\n",
+    "jsonl_object = outputs.to_json_lines()\n",
+    "with Path.open(file_name, \"w\") as f:\n",
+    "    f.write(jsonl_object)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Run evaluate"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "tracking_uri = client.tracking_uri\n",
+    "connection = client.connections.get(\"Default_AzureOpenAI\")\n",
+    "config = AzureOpenAIModelConfiguration.from_connection(connection, model_name=\"gpt-4\", deployment_name=\"gpt-4\")\n",
+    "data_path = Path.cwd() / f\"\\{eval_name}.jsonl\"\n",
+    "output_path = Path.cwd() / \"downloaded_artifacts\" / \"remote\"\n",
+    "result = evaluate(\n",
```

Comment:
similar to previous review, set task_type to "chat" in evaluation function, and update metrics_list.